### PR TITLE
Add config for other LLM

### PR DESCRIPTION
**Pull request by zhouzinimg**
添加了其他大模型的配置方法

### CODE REVIEW

File: src/en/guide/get_started/setup.md [hunk 1]
```diff
@@ -44,12 +44,115 @@ Remember: If you follow the `git clone` approach in [Installation](./installatio
...
+## Visit https://console.cloud.google.com/apis/credentials to get key.
+GOOGLE_API_KEY: "YOUR_API_KEY"
+## Visit https://programmablesearchengine.google.com/controlpanel/create to get id.
+GOOGLE_CSE_ID: "YOUR_CSE_ID"
+
+## Visit https://serper.dev/ to get key.
+SERPER_API_KEY: "YOUR_API_KEY"
+```
+
+#### If it involves accessing website functions
```
Comment (garylin2099) [inline src/en/guide/get_started/setup.md:104]: If you want agents to browse websites

File: src/en/guide/get_started/setup.md [hunk 2]
```diff
@@ -44,12 +44,115 @@ Remember: If you follow the `git clone` approach in [Installation](./installatio
 
 Here you are good to go! See [Quickstart](./quickstart) or our [Tutorials](/guide/tutorials/agent_101) for your first run!
 
-## Setup for different model providers
 
-### OpenAI
+## Detail Setting
+Some future projects may require other API models, so you should configure the keys of other Apis according to your needs.
```
Comment (garylin2099) [inline src/en/guide/get_started/setup.md:49]: MetaGPT supports various LLM models, configure the keys of model APIs according to your needs.

File: src/en/guide/get_started/setup.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 Using MetaGPT involves connecting with model providers. We will walk through the setup in this page.
 
-## Setup for LLM API
+## Fast Setup for LLM API
```
Comment (garylin2099) [inline src/en/guide/get_started/setup.md:5]: Can change to "Setup for OpenAI API"

File: src/en/guide/get_started/setup.md [hunk 4]
```diff
@@ -44,12 +44,115 @@ Remember: If you follow the `git clone` approach in [Installation](./installatio
 
 Here you are good to go! See [Quickstart](./quickstart) or our [Tutorials](/guide/tutorials/agent_101) for your first run!
 
-## Setup for different model providers
 
-### OpenAI
+## Detail Setting
```
Comment (garylin2099) [inline src/en/guide/get_started/setup.md:48]: I recommend maintain the original subheader: Setup for different model providers.
The two sections do not differ in fast or detail but OpenAI API and other APIs

File: src/en/guide/get_started/setup.md [hunk 5]
```diff
@@ -44,12 +44,115 @@ Remember: If you follow the `git clone` approach in [Installation](./installatio
...
+#### You can use ENGINE or DEPLOYMENT mode
+OPENAI_API_TYPE: "azure"
+OPENAI_API_BASE: "YOUR_AZURE_ENDPOINT"
+OPENAI_API_KEY: "YOUR_AZURE_API_KEY"
+OPENAI_API_VERSION: "YOUR_AZURE_API_VERSION"
+DEPLOYMENT_NAME: "YOUR_DEPLOYMENT_NAME"
+DEPLOYMENT_ID: "YOUR_DEPLOYMENT_ID"
+```
+
+#### If you need to use the search function
```
Comment (garylin2099) [inline src/en/guide/get_started/setup.md:85]: If you want agents to search the web

**Comment by zhouzinimg** [comment]
完成。修改了英文文件，并添加中文文件

**Comment by garylin2099** [comment]
LGTM

File: src/en/guide/get_started/setup.md [hunk 6]
```diff
@@ -44,12 +46,117 @@ Remember: If you follow the `git clone` approach in [Installation](./installatio
...
+``` yaml
+#### if Spark
+### Nomally YOU only need to modify SPARK_APPID SPARK_API_SECRET SPARK_API_KEY
+SPARK_APPID : "YOUR_APPID"
+SPARK_API_SECRET : "YOUR_APISecret"
+SPARK_API_KEY : "YOUR_APIKey"
+DOMAIN : "generalv2"
+SPARK_URL : "ws://spark-api.xf-yun.com/v2.1/chat"
+```
+## Baidu's large model  Ernie  API
```
Comment (better629) [inline src/en/guide/get_started/setup.md:66]: there are two spaces, only need one.

File: src/en/guide/get_started/setup.md [hunk 7]
```diff
@@ -44,12 +46,117 @@ Remember: If you follow the `git clone` approach in [Installation](./installatio
...
+#### if zhipuai from `https://open.bigmodel.cn`. You can set here or export API_KEY="YOUR_API_KEY"
+ZHIPUAI_API_KEY: "YOUR_API_KEY"
+```
+## iFlytek's large model Spark API :
+``` yaml
+#### if Spark
+### Nomally YOU only need to modify SPARK_APPID SPARK_API_SECRET SPARK_API_KEY
+SPARK_APPID : "YOUR_APPID"
+SPARK_API_SECRET : "YOUR_APISecret"
+SPARK_API_KEY : "YOUR_APIKey"
```
Comment (better629) [inline src/en/guide/get_started/setup.md:62]: use same `YOUR_API_KEY`. same with the below.

File: src/en/guide/get_started/setup.md [hunk 8]
```diff
@@ -44,12 +46,117 @@ Remember: If you follow the `git clone` approach in [Installation](./installatio
...
+#### You can use ENGINE or DEPLOYMENT mode
+OPENAI_API_TYPE: "azure"
+OPENAI_API_BASE: "YOUR_AZURE_ENDPOINT"
+OPENAI_API_KEY: "YOUR_AZURE_API_KEY"
+OPENAI_API_VERSION: "YOUR_AZURE_API_VERSION"
+DEPLOYMENT_NAME: "YOUR_DEPLOYMENT_NAME"
+DEPLOYMENT_ID: "YOUR_DEPLOYMENT_ID"
+```
+
+## Web Searching API
```
Comment (better629) [inline src/en/guide/get_started/setup.md:89]: belows are not LLM API configuration

File: src/zh/guide/get_started/setup.md [hunk 9]
```diff
@@ -1,55 +1,181 @@
...
 ```python
 import os
+
 os.environ["OPENAI_API_KEY"] = "sk-..."  # YOUR_API_KEY
-os.environ["OPENAI_API_MODEL"] = "intended model" # gpt-4, gpt-3.5-turbo, etc.
+os.environ["OPENAI_API_MODEL"] = "intended model"  # gpt-4, gpt-3.5-turbo, 等等
 ```
 
-### 2. 使用一个`config.yaml`或者`key.yaml`文件
+### 2. 使用key.config
```
Comment (better629) [inline src/zh/guide/get_started/setup.md:33]: key.config ？？

**Comment by better629** [comment]
@zhouzinimg

File: src/en/guide/get_started/setup.md (hunk 8 above)
Comment (zhouzinimg) [reply src/en/guide/get_started/setup.md:89]: 尽管不是，但本章讲的是配置问题。

File: src/en/guide/get_started/setup.md (hunk 6 above)
Comment (zhouzinimg) [reply src/en/guide/get_started/setup.md:66]: 使用PyCharm自动格式化了.

File: src/en/guide/get_started/setup.md [hunk 10]
```diff
@@ -22,34 +26,161 @@ Or in python:
...
+ERNIE_API_KEY : "YOUR_API_KEY"
+ERNIE_SECRET_KEY : "YOUR_API_SECRET"
+```
+
+## Anthropic API
+
+``` yaml
+#### if Anthropic
+Anthropic_API_KEY: "YOUR_API_KEY"
+```
```
Comment (garylin2099) [inline src/en/guide/get_started/setup.md:91]: 还有个疑问，ernie和anthropic现在应该还不支持吧？应该先去掉或者放后面些留空？

File: src/en/guide/get_started/setup.md (hunk 8 above)
Comment (better629) [reply src/en/guide/get_started/setup.md:106]: the above has writen with `MetaGPT also supports various LLM models`. If it's focus on configuration, you can split it with LLM-configuration and non-LLM-configuration. Put them together will make people think it's all about LLM-configuration.

Comment (garylin2099) [reply src/en/guide/get_started/setup.md:106]: 这个我后面提个PR来改吧

**Pull request merged**